tensorflow/python/ops/script_ops.py

```diff
@@ -702,12 +700,16 @@ def numpy_function(func, inp, Tout, name=None):
     inp: A list of `tf.Tensor` objects.
     Tout: A list or tuple of tensorflow data types or a single tensorflow data
       type if there is only one, indicating what `func` returns.
+    stateful: (Boolean.) If True, the function should be considered stateful. If
+      a function is stateless, when given the same input it will return the same
```

Comment:
Please add a comment specifying that if you pass a stateful function with stateless=True, the behavior is undefined.

Reply:
In particular, let's caution about mutating the input arguments.